app/controllers/ex_links_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ExLinksController < ApplicationController
+  before_action :set_ex_link, only: [:show, :edit, :update, :destroy]
+
+  # GET /ex_links
+  # GET /ex_links.json
+  def index
+    @ex_links = ExLink.all
+  end
+
+  # GET /ex_links/1
+  # GET /ex_links/1.json
+  def show
+  end
+
+  # GET /ex_links/new
+  def new
+    @ex_link = ExLink.new
+  end
+
+  # GET /ex_links/1/edit
+  def edit
+  end
+
+  # POST /ex_links
+  # POST /ex_links.json
+  def create
+    @ex_link = ExLink.new(ex_link_params)
+
+    respond_to do |format|
+      if @ex_link.save
+        format.html { redirect_to @ex_link, notice: 'Ex link was successfully created.' }
+        format.json { render :show, status: :created, location: @ex_link }
+      else
+        format.html { render :new }
+        format.json { render json: @ex_link.errors, status: :unprocessable_entity }
```

Comment:
The `respond_to` is useful when there are requests in different formats.
Right now we mostly use the HTML-requeste and therefore do not need the whole `respond_to` block.

See for example: https://github.com/fsek/web/blob/master/app/controllers/notices_controller.rb#L17